src/quadplane_main.py

```diff
@@ -0,0 +1,26 @@
+'''main executable for setting up VTOL'''
+import json
+from util import setup_vehicle
+from quadplane import QuadPlane
+
+def main(configs):
+    '''Configure vtol and ready for mission'''
+    vehicle = setup_vehicle(configs, QuadPlane)
+
+    vehicle.parameters['Q_GUIDED_MODE'] = 1
+    vehicle.takeoff()
+    vehicle.set_altitude(10)
```

Comment:
should this be hard coded?

Reply:
again testing file, I will move the parameter swap to an initialize method though